classworks/Katerina/01_git/result.txt

```diff
@@ -0,0 +1 @@
+git@github.com:ZZireal/summer.git
```

Comment:
@ZZireal This is Please provide a link to the **github repository** (`https://github.com/...`) you've made for solid-train exercise.